modules/wri_event/wri_event.install

```diff
@@ -232,3 +220,41 @@ function wri_event_update_10006() {
 function wri_event_update_10007() {
   \Drupal::service('distro_helper.updates')->installConfig('metatag.metatag_defaults.node__event', 'wri_event', 'install', TRUE);
 }
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function wri_event_update_dependencies() {
+  // Add the new image type first.
+  $dependencies['wri_event'][10501] = [
+    'wri_media' => 10501,
+  ];
+  return $dependencies;
+}
+
+/**
+ * Add a new webinar field to events.
+ */
+function wri_event_update_10501() {
+  // Add the fields.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('field.storage.node.field_webinar', 'wri_event', 'install', 'TRUE');
+
+  // Attach the fields to the Event node.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('field.field.node.event.field_webinar', 'wri_event', 'install', 'TRUE');
+
+  // Updating the Event form display.
+  \Drupal::service('distro_helper.updates')
+    ->updateConfig('core.entity_form_display.node.event.default', [
+      'content#field_webinar',
+    ], 'wri_event');
+
+  // Add the callout display mode.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('core.entity_view_mode.node.callout', 'wri_event', 'install', 'TRUE');
+
+  // Add the callout view display.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('core.entity_view_display.node.event.callout', 'wri_event', 'install', 'TRUE');
+}
```

Comment:
Can you also add an update hook in this install file to enable the `wri_event_callout` module please? A new one, so I can run it on my local.